.github/workflows/buildwebsite.yml

```diff
@@ -0,0 +1,79 @@
+name: continuous build - docs
+
+on: [push, pull_request_target]
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  website-build:
+    runs-on: ubuntu-latest
+    strategy: 
+      fail-fast: false
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Install Linux dependencies
+        run: |
+          sudo apt-get install libopenblas-dev texlive texlive-latex-extra pandoc
+      - name: Setup python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+          architecture: x64
+
+      - name: Install Other Dependencies
+        run: |
+          python -m pip install --quiet --upgrade pip
+          python -m pip install --upgrade cython
+          python -m pip install --pre "mxnet>=2.0.0b20200802" -f https://dist.mxnet.io/python
+          python -m pip install --quiet -e .[extras]
+          python -m pip install setuptools ipython sphinx>=1.5.5 sphinx-gallery nbsphinx notedown sphinx_rtd_theme mxtheme sphinx-autodoc-typehints matplotlib Image recommonmark jupyter_client ipykernel
```

Comment:
what are the sphinx-autodoc-typehints, Image, jupyter_client for? do we need to freeze their versions?

Reply:
`Image` is a package related to image and video resizing and cropping on website, which is suggested in [docs/README](https://github.com/dmlc/gluon-nlp/blob/v0.10.x/docs/README.txt). Some other dependencies like sphinx-autodoc-typehints, jupyter_client are required for compiling notebooks locally. Since, we are compiling notebooks on AWS Batch, I will remove these dependencies.